sdv/constraints/base.py

```diff
@@ -102,6 +102,42 @@ class Constraint(metaclass=ConstraintMeta):
     constraint_columns = ()
     _hyper_transformer = None
 
+    @classmethod
+    def _validate_inputs(cls, **kwargs):
+        errors = []
+        args = []
+        params = inspect.signature(cls).parameters
+        for arg_name, value in params.items():
+            if value.default is inspect._empty:
+                args.append(arg_name)
+
+        missing_values = set(args) - set(kwargs)
+        constraint_name = cls.__name__
+        if missing_values:
+            if constraint_name == 'Inequality':
+                errors.append(ValueError(
+                    f'Missing required values {missing_values} in an {constraint_name} constraint.'
```

Comment:
I don't really like these if statements. @amontanez24 @pvk-developer what do you guys think of deleting it, so the error would just say `a Inequality constraint` instead of `an Inequality constraint`. It doesn't sound too wrong to me, and I'd rather the code be cleaner than a very minor grammatical thing.

Reply:
instead of doing the ifs we can just ake a variable ccalled article and use that in the f-string. eg.
```python
article = 'an' if constraint_name == 'Inequality' else 'a'
errors.append(ValueError(f'Missing required values {missing_values} in {article} {constraint_name} constraint.'))
```